<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_Xlive</class>
 <widget class="QMainWindow" name="main_Xlive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xlive</string>
  </property>
  <widget class="QWidget" name="centralwidget_main_Xlive">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_BeamlineStatus">
     <attribute name="title">
      <string>Beamline status</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>931</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="tab_layout_BeamlineStatus"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ExperimentDesign">
     <attribute name="title">
      <string>Experiment design</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>931</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="tab_layout_ExperimentDesign"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_DataAcquisition">
     <attribute name="title">
      <string>Data acquisition</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar_main_Xlive">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar_main_Xlive"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
